site/app/controllers/admin/ConfigurationController.php

```diff
@@ -46,48 +53,32 @@ public function viewConfiguration()
             'auto_rainbow_grades'            => $this->core->getConfig()->getAutoRainbowGrades()
         );
 
-        if (isset($_SESSION['request'])) {
```

Comment:
This `if` block is removed because I cannot find any usage of `$_SESSION['request']`.

Reply:
Yeah, this is was here back when saving the configuration was a set button and not handled via ajax request.